docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Reordering relations
+description: Use the REST API to manage the order of relations
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/relations-reordering.html
+---
+
+# Managing relations through the REST API
+
+Relations between content-types can be managed through the [admin panel](/user-docs/latest/content-manager/managing-relational-fields.md#managing-multiple-choices-relational-fields) or through REST requests sent to the Content API.
+
+Relations can be added, updated, or removed through the Content API. The `connect`, `disconnect`, and `set` attributes, passed in the body of POST or PUT requests, have different behaviors:
+
+- `connect` and `disconnect` perform partial updates
+- `set` performs a full update, effectively replacing all existing relations
+
+```json
+/** 
+ * Partial update (will add/remove the specified relations)
+ */
+
+// Using the shorthand syntax
+{
+  data: {
+    categories: { connect: [2, 4], disconnect: [5] }
+  }
+}
+
+// Using the longhand syntax
+{
+  data: {
+    categories: { connect: [{ id: 2 }, { id: 4 }], disconnect: [{ id: 5 }] }
+  }
+}
+
+/**
+ * Full update (will delete all current relations and set those ones instead)
+ */
+
+// Using the shorthand syntax
+{
+  data: {
+    categories: { set: [2, 4] }
+  }
+}
+
+// Using the longhand syntax
+{
+  data: {
+    categories: { set: [{ id: 2 }, { id: 4 }] }
+  }
+}
+```
```

Comment:
Thanks! I think I wouldn't add it here as it could confuse users. The goal was to highlight connect/disconnect vs. set.
But I will add it to the set-related section down the page, thank you!